### ISSUE TITLE
HTTP --> HTTPS redirect

### ISSUE THREAD

**Issue by LindseyWeiskopf** [googleinterns/SmallCity#18]
Find way to keep browser from automatically blocking location services when initiated with an http connection or default step2020-smallcity.appspot.com

**Comment by LindseyWeiskopf** [comment]
I found a solution to the issue involving modifying the web.xml file under the target folder to allow confidential protocol on the site. Now, I am trying to figure out if this is a change that should be pushed to github or added locally.

**Comment by LindseyWeiskopf** [comment]
The change was pushed.

Other solutions attempted:

- Alter the app.yaml file in the target folder such that 'secure: always' is listed under every url entry. This did not work because the app.yaml file gets overridden each time the code is run. The current solution autimatically makes these changes to the default app.yaml yielding the same result.

- Investigated SSL certificates on GAE -- I realized that all appspot applications are initiated with an SSL certificate

Other potential alternatives that could have been implemented:

- Page redirect via Javascript - the CONFIDENTIAL method in the config document ensures that the underlying connected is secure as opposed to taking the extra step of redirecting the user.
